### PR TITLE
Add Drupal, Grav, Nextcloud and Python as installable/creatable Apps

### PR DESCRIPTION
**Pull request by niristius**

### CODE REVIEW

File: src/commands/app/create/python.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+} from "../../../lib/app/Installer.js";
+
+export const staticInstaller = new AppInstaller(
+  "be57d166-dae9-4480-bae2-da3f3c6f0a2e",
+  "custom python site",
+  ["document-root", "site-title", "wait"] as const,
+);
+
+export default class InstallNode extends ExecRenderBaseCommand<
+  typeof InstallNode,
```
Comment (martin-helmich) [inline src/commands/app/create/python.tsx:15]:
```suggestion
export default class InstallPython extends ExecRenderBaseCommand<
  typeof InstallPython,
```

File: src/commands/app/create/python.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import { ExecRenderBaseCommand } from "../../../rendering/react/ExecRenderBaseCommand.js";
+import React from "react";
+import {
+  AppInstallationResult,
+  AppInstaller,
+} from "../../../lib/app/Installer.js";
+
+export const staticInstaller = new AppInstaller(
```
Comment (martin-helmich) [inline src/commands/app/create/python.tsx:8]:
```suggestion
export const pythonInstaller = new AppInstaller(
```

File: src/commands/app/create/python.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+      this.apiClient,
+      this.args,
+      this.flags,
+      this.config,
+    );
+  }
+
+  protected render(result: AppInstallationResult): React.ReactNode {
+    return staticInstaller.render(result, this.flags);
+  }
```
Comment (martin-helmich) [inline src/commands/app/create/python.tsx:32]:
```suggestion
  static description = pythonInstaller.description;
  static flags = pythonInstaller.flags;

  protected async exec(): Promise<{ appInstallationId: string }> {
    return pythonInstaller.exec(
      this.apiClient,
      this.args,
      this.flags,
      this.config,
    );
  }

  protected render(result: AppInstallationResult): React.ReactNode {
    return pythonInstaller.render(result, this.flags);
  }
```

File: src/commands/app/install/drupal.tsx [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    "admin-user",
+    "admin-email",
+    "admin-pass",
+    "site-title",
+    "wait",
+  ] as const,
+);
+
+export default class InstallShopware6 extends ExecRenderBaseCommand<
+  typeof InstallShopware6,
```
Comment (martin-helmich) [inline src/commands/app/install/drupal.tsx:23]:
```suggestion
export default class InstallDrupal extends ExecRenderBaseCommand<
  typeof InstallDrupal,
```

File: src/commands/app/install/nextcloud.tsx [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+    "admin-user",
+    "admin-email",
+    "admin-pass",
+    "site-title",
+    "wait",
+  ] as const,
+);
+
+export default class InstallShopware6 extends ExecRenderBaseCommand<
+  typeof InstallShopware6,
```
Comment (martin-helmich) [inline src/commands/app/install/nextcloud.tsx:23]:
```suggestion
export default class InstallNextcloud extends ExecRenderBaseCommand<
  typeof InstallNextcloud,
```

**Review by martin-helmich — CHANGES_REQUESTED** [review]
LGTM in general, but I've found some copy&paste issues; also, please run `yarn generate:readme` to update the README.

**Comment by niristius** [comment]
I guess I wasn't awake by then, goddamnit.
Fixed the copy and paste errors in my latest commit.
I also took the liberty of fixing some inherited copy and paste errors in the commands/app/create/static.tsx from wich I copied the python.tsx. It was kinda confusing that I oversaw mentions of "static" aswell as "node" - apologies!

File: src/commands/app/create/static.tsx [hunk 6]
```diff
@@ -11,8 +11,8 @@ export const staticInstaller = new AppInstaller(
   ["document-root", "site-title", "wait"] as const,
 );
 
-export default class InstallNode extends ExecRenderBaseCommand<
-  typeof InstallNode,
+export default class InstallStatic extends ExecRenderBaseCommand<
+  typeof InstallStatic,
```
Comment (martin-helmich) [inline src/commands/app/create/static.tsx:15]: Yep, good catch. :+1:

**Review by martin-helmich — APPROVED** [review]

**Pull request merged**